source/guide_bitwarden.rst

```diff
@@ -207,9 +207,15 @@ You can create a backup of the database manually. ``cd`` to your project folder,
 
  [isabell@stardust ~]$ cd ~/bitwarden_rs/data
  [isabell@stardust data]$ mkdir db-backup
- [isabell@stardust data]$ sqlite3 ~/bitwarden_rs/data/db.sqlite3 ".backup '~/bitwarden_rs/data/db-backup/backup.sqlite3'"
+ [isabell@stardust data]$ sqlite3 ~/bitwarden_rs/data/db.sqlite3 ".backup '$HOME/bitwarden_rs/data/db-backup/backup.sqlite3'"
```

Comment:
I am not sure what's going on here. `~` is replaced by the shell, not sqlite so this should work fine. It also seems like it does:

```
[root@stardust ~]# echo sqlite3 ~/bitwarden_rs/data/db.sqlite3 ".backup '$HOME/bitwarden_rs/data/db-backup/backup.sqlite3'"
sqlite3 /root/bitwarden_rs/data/db.sqlite3 .backup '/root/bitwarden_rs/data/db-backup/backup.sqlite3'
```

Which client and which shell are you using?

Reply:
I am using Windows Terminal with Powershell. It is weird that ~ outside the quote can be replaced correctly but not the one inside the quote. Also tried WSL and bash on Mint Linux, still no luck, always got the same error:

```
[user@adrastea ~]$ sqlite3 ~/bitwarden_rs/data/db.sqlite3 ".backup '~/bitwarden_rs/data/db-backup/$(date +%Y-%m-%d).sq
lite3'"
Error: cannot open "~/bitwarden_rs/data/db-backup/2021-03-04.sqlite3"
```

With $HOME instead I could create backups successfully. 

I am not sure whether ~ is interpreted by sqlite3 as a database operator before it gets replaced by shell?